app/models/assign_to.rb

```diff
@@ -3,7 +3,7 @@ class AssignTo < ApplicationRecord
   belongs_to :gestionnaire
   has_one :procedure_presentation, dependent: :destroy
 
-  def procedure_presentation_or_default
-    procedure_presentation || build_procedure_presentation
+  def procedure_presentation_or_default_with_errors
```

Comment:
Idem, nom pas évident, `procedure_presentation_or_default_and_errors` (c'est peut-être plus flou comme nom cela dit) ?

Reply:
remplacer le `and` par un `with` ? Moi ça me va.

Reply:
Pour clarifier : c'est l'inverse que je propose ici, remplacer dans ton code le `with` par un `and`

Reply:
heu oui, je me suis emmêlé les pinceaux. En tout cas c’est fait.